.vscode/tasks.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Update cr/crd files",
+            "command": "olm/./update-cr-crd-files.sh",
```

Comment:
It was my mistake to name it like `update-cr-crd-files.sh`
It actually updates only crd files.
Could you rename it accordingly ? : `update-crd-files.sh`